ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16.yaml

```diff
@@ -68,7 +68,7 @@ images:
         tar -C /usr/bin/ -xzvf openshift-client-linux.tar.gz && rm -f openshift-client-linux.tar.gz && \
         OCP_VERSION="$(/usr/bin/oc version --client | grep 'Client Version' | cut -d':' -f2 | xargs)" && \
         curl -skL ${MIRRORURL}/ocp/stable-4.16/opm-linux-rhel9.tar.gz -o opm-linux.tar.gz  && \
-        tar -C /usr/bin/ -xzvf opm-linux.tar.gz && rm -f opm-linux.tar.gz && \
+        tar -C /usr/bin/ -xzvf opm-linux.tar.gz && mv /usr/bin/opm-rhel9 /usr/bin/opm && opm version && && rm -f opm-linux.tar.gz && \
```

Comment:
`&& && ` is unexpected.

Reply:
Thanks. update it.
I wait rehearse job pass and then request to merge it.